server/neptune/workflows/internal/root/root.go

```diff
@@ -1,33 +1,37 @@
 package root
 
 import (
-	"path/filepath"
-
 	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/github"
 	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/job"
 )
 
+// Root is the definition of a root
 type Root struct {
-	Name      string
+	Name string
+
+	// Path is the relative path from the repo
 	Path      string
```

Comment:
nit: should we be more explicit and name it relPath to avoid confusion when accessing the fields?

Reply:
Sure, but what other path would it represent?